app/views/users/_profilenav.html.erb

```diff
@@ -0,0 +1,25 @@
+<h1>
+  <%= @user.username %>
+</h1>
+
+<div class="row">
+  <div class="col md-6 offset-3">
```

Comment:
should be `col-md-6 offset-3`

Reply:
need to add a hyphen after `col`
Currently it's `col md-6` but should be `col-md-6`